usaspending_api/submissions/management/commands/load_submission.py

```diff
@@ -309,22 +301,67 @@ def handle(self, *args, **options):
         self.logger.info('Acquired award procurement data for ' + str(submission_id) + ', there are ' + str(len(procurement_data)) + ' rows.')
 
         for row in procurement_data:
-            # Yes, I could use Q objects here but for maintainability I broke it out
-            award = Award.objects.filter(piid=row['parent_award_id']).first()
-            if award is None:
-                award = Award.objects.filter(fain=row['parent_award_id']).first()
-            if award is None:
-                award = Award.objects.filter(uri=row['parent_award_id']).first()
-            if award is None:
-                self.logger.error('Could not find an award object with a matching identifier: ' + row['parent_award_id'])
-                continue
+            # Try and find the parent award
+            award = None
+            try:
+                award = find_or_create_summary_award(piid=row.get('piid'), fain=row.get('fain'), uri=row.get('uri'), parent_award_id=row.get('parent_award_id'))
```

Comment:
Recording this for posterity... @afrasier is correct about `parent_award_id` in this context. I was getting confused because I'm used to the term `parent` being used in conjunction with sub-awards. However, parent refers to something different in this context:

```
 in the schema, when a contract is a Delivery Order, BPA Call, or Purchase Order, it would be attached to a parent ID (or contract vehicle, i.e. Federal Supply Schedule, GWAC, etc)
```
in other words, I this should be good to go...thanks for accommodating my confusion, @afrasier!